aws-dynamodb-point-in-time-recovery-disabled.yaml

```diff
@@ -0,0 +1,37 @@
+rules:
+- id: aws-dynamodb-point-in-time-recovery-disabled
+  patterns:
+  - pattern: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+      }
+  - pattern-not-inside: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+        point_in_time_recovery {
+          enabled = true
+        }
+        ...
+      }
+  message: Point-in-time recovery is not enabled for the DynamoDB table. DynamoDB
+    tables should be protected against accidental or malicious write/delete actions.
+    By enabling point-in-time-recovery you can restore to a known point in the event
+    of loss of data.
+  languages:
+  - hcl
+  severity: WARNING
+  metadata:
+    category: security
+    technology:
+    - terraform
+    - aws
+    cwe:
+    - 'CWE-221: Information Loss or Omission'
+    subcategory:
+    - audit
+    likelihood: LOW
+    impact: LOW
+    confidence: LOW
+    rule-origin-note: published from /src/aws-dynamodb-point-in-time-recovery-disabled.yml
+      in None
+    owasp: A9:2021 bad-logging
```

Comment:
```suggestion
    owasp: 
     - A09:2021 – Security Logging and Monitoring Failures
```

Reply:
Hmm, this comes from selecting the OWASP value in the semgrep app?

<img width="646" alt="Screenshot 2023-02-23 at 11 25 17 AM" src="https://user-images.githubusercontent.com/5559014/220968362-f447897f-a317-4d9d-9784-5ca5e9be35b5.png">